### PR TITLE
Add Kubernetes boilerplate to go files

### PR DESCRIPTION
**Pull request by pmorie**
... to facilitate donation to kubernetes-sigs

### CODE REVIEW

File: v2/auth_test.go [hunk 1]
```diff
@@ -1,3 +1,16 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```
Comment (nikhita) [inline v2/auth_test.go]: oops needs some more newlines here:

https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt

**Comment by pmorie** [comment]
@nikhita ptal

File: v2/fake/fake.go [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+
 package fake
 
 import (
 	"errors"
 	"net/http"
 	"sync"
 
-	"github.com/pmorie/go-open-service-broker-client/v2"
+	v2 "github.com/pmorie/go-open-service-broker-client/v2"
```
Comment (nikhita) [inline v2/fake/fake.go]: Looks like this isn't necessary...

Comment (pmorie) [reply v2/fake/fake.go]: lol, not sure where that came from

**Comment by nikhita** [comment]
@pmorie looks like this is good to go (pending the v2 import path changes). Will migrate the repo once this has merged. :)

**Comment by nikhita** [comment]
Thanks :)

**Pull request merged**